src/components/button/index.tsx

```diff
@@ -290,6 +291,11 @@ class Button extends PureComponent<Props, ButtonState> {
       if (typeof iconSource === 'function') {
         return iconSource(iconStyle);
       } else {
+        if (Constants.isWeb) {
```

Comment:
Yes, if we will user Icon and the user don't pass any `tintColor` the Icon will get the default color.
So if we change it to Icon it will break some users.

Reply:
Ok, we need to migrate to Icon IMO, we can do it v8. 
Can you add it to the v8 spreadsheet

Reply:
Done added to the spreadsheet.